src/js/protocols/stm32.js

```diff
@@ -85,49 +88,120 @@ STM32_protocol.prototype.connect = function (port, baud, hex, options, callback)
             }
         });
     } else {
-        serial.connect(port, {bitrate: self.options.reboot_baud}, function (openInfo) {
-            if (openInfo) {
-                console.log('Sending ascii "R" to reboot');
 
-                // we are connected, disabling connect button in the UI
-                GUI.connect_lock = true;
+        var startFlashing = function() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Combine this with the previous 'var' statement.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5626707)